src/main/java/jenkins/scm/api/SCMFileSystem.java

```diff
@@ -144,7 +148,11 @@ public static SCMFileSystem of(@NonNull SCMSource source, @NonNull SCMHead head,
                                    @CheckForNull SCMRevision rev) {
         source.getClass(); // throw NPE if null
         SCMFileSystem fallBack = null;
-        for (Builder b : Jenkins.getInstance().getExtensionList(Builder.class)) {
+        Jenkins j = Jenkins.getInstance();
+        if (j == null) {
+            return fallBack;
```

Comment:
Ditto, this was just to shut up FindBugs and can be simplified further with a new core.